# Automatically Generated in 2017-06-12 22:33:47.453220
# MESSAGE: CONFIG_PARAM_SET_BOOL
# Description:Set new config parameter bool value
uint8 ID = 201
uint8 sysid
uint8 compid
uint8 config_id
uint8 param_id
uint8 value
